Time/DayAndNightClock

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# <bitbar.title>DayAndNightClock</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>FusionX</bitbar.author>
+# <bitbar.author.github>FusionX13</bitbar.author.github>
+# <bitbar.desc>A clock that can do many things.</bitbar.desc>
+# <bitbar.image>https://ibb.co/CBLByKf</bitbar.image>
```

Comment:
Should be:

```
# <bitbar.image>https://i.ibb.co/6YQYS0k/Screen-Shot-2020-04-24-at-11-38-18-AM.png</bitbar.image>
```

Reply:
Ok I did that part. By the way, thanks for providing that.